apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,240 @@
+<!-- JavaScript code -->
+<script lang='ts'>
+  //Script to obtain current year
+  const currentYear = (new Date().getFullYear() as unknown) as string;
+
+  //Script to obtian Previous year
+  const previousYear = ((new Date().getFullYear() - 1) as unknown) as string;
+  
+</script>
+
+<!-- Landing Page - Card -->
+<div class="flex flex-row items-center justify-center h-screen">
+
+    <!-- Landing Page - Content -->
+    <div class=" w-full max-w-7xl h-screen p-10 rounded-md">
+        
+        <!-- Landing Page - Header -->
+        <div class="dropdown dropdown-bottom">
+            <div tabindex="0" role="button" class="btn m-1 text-2xl font-semibold normal-case px-6 rounded-full border-none bg-transparent">My Files
+                <span>
+                    <svg xmlns="http://www.w3.org/2000/svg" height="20" width="14" viewBox="0 0 320 512" fill="#fff">
+                        <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z"/>
+                    </svg>
```

Comment:
Save this `svg` into a file of it's own and use it dynamically via imports.

Reply:
Didn't use imports, instead simply added svg files into static and used an `<img>` tag. Let me know if this is okay.

![image](https://github.com/taronaeo/IT2566-Armadillo/assets/125195946/2070c301-52cc-4e13-b59b-d664df76bb3e)